chrome/fed-modules.json

```
@@ -623,15 +623,27 @@
         ]
     },
     "sandbox": {
-      "manifestLocation": "/apps/sandbox/fed-mods.json",
-      "modules": [
+        "manifestLocation": "/apps/sandbox/fed-mods.json",
+        "modules": [
+            {
+                "id": "sandbox",
+                "module": "./RootApp",
+                "routes": [
+                    "/openshift/sandbox"
+                ]
+            }
+      ]
+    },
+    "acs": {
+        "manifestLocation": "/apps/acs/fed-mods.json",
+        "modules": [
           {
-              "id": "sandbox",
+              "id": "acs",
               "module": "./RootApp",
               "routes": [
-                  "/openshift/sandbox"
+                  "/acs"
```

This should include the full path
```suggestion
                  "/application-services/acs"
```